pkg/server/commands/listusers/list_users_rpc.go

```diff
@@ -487,18 +486,19 @@ func (l *listUsersQuery) expandExclusion(
 	baseFoundUsersCh := make(chan *openfgav1.User, 1)
 	subtractFoundUsersCh := make(chan *openfgav1.User, 1)
 
-	var errs error
+	var errsBase error
+	var errsSub error
```

Comment:
Nit: not plural, right?

```suggestion
	var errBase error
	var errSub error
```